src/Shared/ProblemDetails/ProblemDetailsDefaults.cs

```diff
@@ -63,11 +76,35 @@ internal static class ProblemDetailsDefaults
             "Unprocessable Entity"
         ),
 
+        [426] =
+        (
+            "https://tools.ietf.org/html/rfc9110#section-15.5.22",
+            "Upgrade Required"
+        ),
+
         [500] =
         (
-            "https://tools.ietf.org/html/rfc7231#section-6.6.1",
+            "https://tools.ietf.org/html/rfc9110#section-15.6.1",
             "An error occurred while processing your request."
         ),
+
+        [502] =
```

Comment:
I was hoping to be able to consolidate both lists (`ProblemDetailsDefaults.Defaults` and `ReasonPhrases.Phrases` , including the `reference URL`, to avoid this adding new status codes here. But we probably will need a public API change :(.